exercises/e001/results/mikjarz/e001.py

```diff
@@ -0,0 +1,30 @@
+import random
+import base64
+import argparse
+
+parser = argparse.ArgumentParser(description="Random string of base32 characters")
+parser.add_argument('characters', default=32, type=int, nargs='?', help='How many characters to print, default=32')
+caseGroup = parser.add_mutually_exclusive_group()
+parser.add_argument('-u', '--upper', action='store_true', help='Printing all upper characters')
+parser.add_argument('-l', '--lower', action='store_true', help='Printing all lower characters')
```

Comment:
I think you could store result in the same variable for both options and for `-u` do `action='store_true'` and for `-l` do `action='store_false'`. You end up with only one variable to check, I think it's simpler.

Reply:
You need common variable for both options. You need to read `argparse` documentation how to define custom target variable for an option.

Reply:
I'm not sure If I know what you mean. Since I can use only one option at one time (either -u or -l), why would I make them into one variable? Additionally, how can I make them into one variable, If they are in one parser? If they were in seperate parsers, I could use `(parents=[parent_parser])` object, but in this case I don't think that it makes sense.

Reply:
https://stackoverflow.com/questions/14110458/set-default-value-with-argparse-and-same-dest-attribute

`dest=attrname` and `store_true` or `store_false`